src/reporters/lang/en.js

```diff
@@ -238,17 +238,16 @@ const messages = {
   errorDecompressingTarball: '$0. Error decompressing $1, it appears to be corrupt.',
   updateInstalling: 'Installing $0...',
   hostedGitResolveError: 'Error connecting to repository. Please, check the url.',
-
+  
   unknownFetcherFor: 'Unknown fetcher for $0',
-
+  
   refusingDownloadGitWithoutCommit: 'Refusing to download the git repo $0 over plain git without a commit hash',
   refusingDownloadHTTPWithoutCommit: 'Refusing to download the git repo $0 over HTTP without a commit hash',
   refusingDownloadHTTPSWithoutCommit: 'Refusing to download the git repo $0 over HTTPS without a commit hash - possible certificate error?',
-
+  
```

Comment:
Could you please revert all the unnecessary changes, such as these spaces? In general, the Facebook coding style (which is somewhat used in the Yarn codebase too) is to trim all trailing whitespace from every line, so blank lines like these should be totally blank.